python/lsst/meas/base/sdssShape.cc

```diff
@@ -40,6 +40,32 @@ namespace afwGeom = lsst::afw::geom;
 namespace lsst {
 namespace meas {
 namespace base {
+namespace {
+
+using PySdssShapeAlgorithm = py::class_<SdssShapeAlgorithm, SimpleAlgorithm>;
```

Comment:
Unintentional.  But I'm guessing if I add it here and not in `Algorithm` and all of its derived classes, I'll get segfaults, right?  If so I'd rather defer this to the cleanup issue.